runtime/compiler/runtime/RuntimeAssumptions.cpp

```diff
@@ -328,12 +333,10 @@ TR_PersistentCHTable::removeClass(
 
       }
 
-   if (!removeInfo)
-      info->setFirstSubClass(0);
-   else
+   if (removeInfo)
       {
       _classes[hashPos].remove(info);
-      info->removeSubClasses();
+      // info->removeSubClasses();
```

Comment:
We removed `info->removeSubClasses()` in jitaas. I need to investigate why we did it.

Reply:
```
void
TR_PersistentClassInfo::removeSubClasses()
   {
   _classId = 0;
   _subClasses.setFirst(0);
   }
```
does what `info->setFirstSubClass(NULL);`
so i think this change is safe